src/caffe/proto/caffe.proto

```diff
@@ -306,7 +306,7 @@ message ParamSpec {
 // NOTE
 // Update the next available ID when you add a new LayerParameter field.
 //
-// LayerParameter next available layer-specific ID: 147 (last added: recurrent_param)
+// LayerParameter next available layer-specific ID: 148 (last added: recurrent_param)
```

Comment:
I do't think too much about that.If I can't modified it,you can rewrite it.sorry